### ISSUE TITLE
Release 6.5.0

### ISSUE THREAD

**Issue by hackmastera** [projecthydra/sufia#1426]
continuing a conversation from projecthydra/sufia#1402. @mjgiarlo @awead 

Steps:
1. Create 6.x-stable from 6.0-stable
2. Delete 6.0-stable
4. Create 6.4.1 branch from 6.4.0
5. PR a backport of the commit in question to 6.4.1.
6. Mint a 6.4.1 release (presumably this is its own set of steps with which I lack familiarity)

These are mostly things that must be done directly on this repo (i.e. cannot be done via PR).

I await confirmation of this plan! Let me know if you want me to attempt this stuff or you want to take care of it.

**Comment by mjgiarlo** [comment]
Here are the instructions for step 6, in the meantime: https://github.com/projecthydra/sufia/wiki/Release-management-process

Mulling over the broader proposal now. Gimme a minute or two.

**Comment by mjgiarlo** [comment]
@hackmastera I'm flexible here, but a 6.5.0 release does not look much more substantial than a 6.4.1 release. The only extra step is recording in the release notes that there's one new generator to run for a clean upgrade path, and it creates a config variable that allows the ability to tweak collection faceting.

See here for the diff between v6.4.0 and 6.0-stable HEAD: https://github.com/projecthydra/sufia/compare/v6.4.0...6.0-stable

So, what would you think about this friendly amendment:

1. Rename `6.0-stable` to `6.x-stable`
2. Push `6.x-stable` to upstream (projecthydra/sufia)
3. Delete `6.0-stable` from upstream
4. Follow the release management process for cutting 6.5.0

:question: 

I am happy to help with the process!

**Comment by mjgiarlo** [comment]
I took the liberty of starting down this path by renaming the branch. See the struckthrough steps in the prior comment. (And, btw, you can start the release mgmt process at its step 4.)

**Comment by hackmastera** [comment]
@mjgiarlo I think you pushed from an outdated copy of the branch -- I'm showing some missing commits which I will try to restore

| *   e82c99e (upstream/6.0-stable) Merge pull request #1412 from hackmastera/backport-fix1402 [Justin Coyne, 2015-12-09] [Justin Coyne, 2015-12
-09]
| |\  
| | * aae4ca4 (origin/backport-fix1402, backport-fix1402) Remove unnecessary require [Justin Coyne, 2015-12-08] [Anna Headley, 2015-12-09]
| |/  
| *   ead1c0a Merge pull request #1363 from projecthydra/collection_facets [Michael J. Giarlo, 2015-12-01] [Michael J. Giarlo, 2015-12-01]
| |\  
| | * c779f78 Option to show collection facet in Dashboard and search results [E. Lynette Rayle, 2015-11-24] [E. Lynette Rayle, 2015-11-30]
| |/  
| *   8aa7086 (HEAD, upstream/6.x-stable, 6.x-stable, 6.0-stable) Merge pull request #1355 from hackmastera/autolink-icon-fix [Michael J. Giarlo, 2015-11-24] [Michael J. Giarlo, 2015-11-24]

**Comment by hackmastera** [comment]
Okay the rake task didn't go through. and 'gem push' (not entirely sure what that is supposed to do but I tried it anyway) tells me 'Access denied'. I was able to set up a ~/.gem/credentials file so I'm accessing the rubygems site itself okay. But then when I do gem push after setting up that api key it looks like it wants another argument. Anyway here's the output.

$ gem push; rake all:release
ERROR:  While executing gem ... (Gem::CommandLineError)
    Please specify a gem name on the command line (e.g. gem build GEMNAME)
rm -f pkg/sufia-models-6.5.0.gem
cd sufia-models && gem build sufia-models.gemspec && mv sufia-models-6.5.0.gem /Users/aheadley/src/actual-sufia/pkg/
WARNING:  description and summary are identical
WARNING:  open-ended dependency on mini_magick (>= 0) is not recommended
  if mini_magick is semantically versioned, use:
    add_runtime_dependency 'mini_magick', '~> 0'
WARNING:  open-ended dependency on posix-spawn (>= 0) is not recommended
  if posix-spawn is semantically versioned, use:
    add_runtime_dependency 'posix-spawn', '~> 0'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: sufia-models
  Version: 6.5.0
  File: sufia-models-6.5.0.gem
rm -f pkg/sufia-6.5.0.gem
gem build sufia.gemspec && mv sufia-6.5.0.gem /Users/aheadley/src/actual-sufia/pkg/
WARNING:  open-ended dependency on railties (>= 4.2.0) is not recommended
  if railties is semantically versioned, use:
    add_runtime_dependency 'railties', '~> 4.2', '>= 4.2.0'
WARNING:  prerelease dependency on tinymce-rails-imageupload (~> 4.0.16.beta) is not recommended
WARNING:  open-ended dependency on sass-rails (>= 0) is not recommended
  if sass-rails is semantically versioned, use:
    add_runtime_dependency 'sass-rails', '~> 0'
WARNING:  pessimistic dependency on breadcrumbs_on_rails (~> 2.3.0) may be overly strict
  if breadcrumbs_on_rails is semantically versioned, use:
    add_runtime_dependency 'breadcrumbs_on_rails', '~> 2.3', '>= 2.3.0'
WARNING:  pessimistic dependency on select2-rails (~> 3.5.9) may be overly strict
  if select2-rails is semantically versioned, use:
    add_runtime_dependency 'select2-rails', '~> 3.5', '>= 3.5.9'
WARNING:  open-ended dependency on json-schema (>= 0) is not recommended
  if json-schema is semantically versioned, use:
    add_runtime_dependency 'json-schema', '~> 0'
WARNING:  open-ended dependency on oauth (>= 0) is not recommended
  if oauth is semantically versioned, use:
    add_runtime_dependency 'oauth', '~> 0'
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: sufia
  Version: 6.5.0
  File: sufia-6.5.0.gem
git add . && git commit --verbose --template=pkg/commit_message.txt
On branch 6.x-stable
nothing to commit, working directory clean
rake aborted!
Command failed with status (1): [git add . && git commit --verbose --templa...]
/Users/aheadley/src/actual-sufia/tasks/release.rake:81:in `block (2 levels) in <top (required)>'
Tasks: TOP => all:release => all:commit
(See full trace by running task with --trace)
